### DISCUSSION TITLE
Is Slingbox still attempting to brick boxes?

### DISCUSSION THREAD

**Discussion by skijohn877** [GerryDazoo/Slinger#575] [General]
Hello All,

Wondering if Slingbox is still attempting to brick devices. I just upgraded my FIOS router to a G3100. There apparently is no way to block traffic inbound/outbound. I have UPNP turned off and always set up port forwarding manually for any devices. So far port 5678 is not accessible from outside my LAN. Wondering if this would be sufficient.

Thanks

**Comment by GerryDazoo** [comment]
Wondering if Slingbox is still attempting to brick devices.
    Yes, if you have a ProHD.
 I just upgraded my FIOS router to a G3100. There apparently is no way to block traffic inbound/outbound. 
   Yes there is. Use the Parental Controls documenting chapters 4/5 in your manual, to block internet access for your slingbox.

So far port 5678 is not accessible from outside my LAN. Wondering if this would be sufficient.
      No it's port 5678 OUTGOING from your LAN not incoming that needs to be blocked.

**Reply by skijohn877** [reply]
Thanks, so it's only the ProHD that are still at risk? I have a couple of 350's that are working great. But, just to be on the safe side, I blocked them (at least I think I did) from internet access.

**Reply by GerryDazoo** [reply]
So far just the ProHD but we've been recommending blocking for all boxes just in case.

**Reply by mike-uk** [reply]
@GerryDazoo I thought Solo and Pro are still at risk? Do you mean they have turned of the kill switch command for Solo and Pro now?

**Reply by GerryDazoo** [reply]
Yes the Solos and Pros will get bricked.

**Reply by mike-uk** [reply]
@GerryDazoo Does all models before Pro-HD get bricks? And Pro-HD is the only model from the lots that can be unbricked? This would be useful information to put at the beginning of readme.
